src/components/help-modal/index.tsx

```diff
@@ -21,24 +21,32 @@ const HelpModal = () => {
           <kbd>Alt</kbd> + <kbd> Shift </kbd> + <kbd> f </kbd> / <kbd>&#8997;</kbd> + <kbd>&#8984;</kbd> + <kbd>f</kbd>:
           Auto format
         </li>
+        <li>
+          <kbd>Ctrl</kbd> + <kbd> f11 </kbd> / <kbd>&#8984;</kbd> + <kbd>f11</kbd>: Toggle Fullscreen Mode
+        </li>
       </ul>
       <a href="https://github.com/vega/editor/issues/new" target="_blank" className="report-button">
         <AlertCircle className="header-icon" />
         Report a Bug{' '}
       </a>
-      <br />
       <h2>Have a feedback? Let us know</h2>
-      <a href="https://bit.ly/join-vega-slack" target="_blank">
+      <a href="https://bit.ly/join-vega-slack" target="_blank" className="link-to-docs">
+        <img src="/images/utils/slack-logo-icon.png" className="doc-image" />
```

Comment:
Please use https://feathericons.com/ like we do in other places (using react feather).